coding-intro/coding-intro-core/core-concepts/organizing-commands.md

```diff
@@ -7,35 +7,24 @@ category: must-know
 
 ---
 
-# Organizing Commands
+# Functions
 
 ---
 ## Content
 
-Functions allow us to group commands and create new ones.
+Functions allow us to make little snippets of code, each for a separate task that we do within our program. For instance, we are simulating a human being that loves coffee, so much that he makes 10 cups a day. For every cup of coffee he would need to do the following:
 
-### Organizing behavior
+1. find a clean cup
+2. boil water
+3. open the packet
+4. mix the coffee with the boiling water.
+5. drink (gulp, gulp)
 
-Using functions we can organize commands into groups and give each group a name.
-
-For example, if you had a program that was meant to simulate a person going for a run, you might have a function called `goForARun` that makes use of the following commands:
-- lift left leg
-- lift right leg
-- inhale
-- exhale
-
-The function `goForARun` would now be a new command in your program that consists of 4 subcommands.
-
-### Sharing functionality
-
-New commands created with functions can be used anywhere in the program, many times if needed.
-
-For example, you could use the command `goForARun` anytime your user presses the controls to speed up their game character.
+That's so much work! Instead, imagine if we had one function `makeCoffeee` with all of these instructions. Every time we want coffee, we write `makeCoffee` and, voila! - the work is done.
 
 ### Easier to read
 
-Just like organizing a book by chapters makes the book easier to read, so does organizing your program into functions.
-
-Functions give the program structure.
+Functions give the program structure and make them easy to understand. Makebed, BrushTeeth, and ReadBook would be a lot easier to understand rather than a series of steps laid out one after the other.
```

Comment:
```suggestion
Functions give the program structure and make them easy to understand. MakeBed, BrushTeeth, and ReadBook would be a lot easier to understand rather than a series of steps laid out one after the other.
```

Reply:
Although I guess we should use snake_case? Not sure cc @kapnobatai137

Reply:
Wasn't using them in code blocks so i thought this might make them more readable. But if snake_case would be more suitable can convert them. Up to you guys.

Reply:
yeah, we use snake_case for function names throughout the topic, so we should use them here as well

```suggestion
Functions give the program structure and make them easy to understand. `make_bed`, `brush_teeth`, and `read_book` would be a lot easier to understand rather than a series of steps laid out one after the other.
```